### ISSUE TITLE
Insufficient contrast on light gray vertical bars on page menus

### ISSUE THREAD

**Issue by lpgoldstein** [DOI-ONRR/nrrd#2720]
![image.png](https://images.zenhubusercontent.com/5c59d3cd7592186396262002/ea00f8b1-8e2b-4081-b3d1-4c4e881c5d07)
